docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - traefik.http.services.frontend.loadbalancer.server.port=80
 
   postgresql:
-    image: mdillon/postgis:11
+    image: postgis/postgis:11-3.1
```

Comment:
We can now safely use [postgis/postgis:14-3.3](https://registry.hub.docker.com/r/postgis/postgis/tags?page=1&name=14-3.3)